news/4664.hooks.rst

```diff
@@ -0,0 +1 @@
+Add hook for `Uvicorn <https://www.uvicorn.org/>`_ (`#4663 <https://github.com/pyinstaller/pyinstaller/issues/4663>`_) 
```

Comment:
Please remove the reference to #4663 and *copy* this file to `4663.hooks.rst`. The release will then create the appropriate references automatically.

Reply:
Done as requested